### PR TITLE
In theory, this fixes the building issues.

### PR DESCRIPTION
**Pull request by amboyscout**

### CODE REVIEW

**Comment by amboyscout** [comment]
This was accidental. Seems to have worked though.

File: pom.xml [hunk 1]
```diff
@@ -9,12 +9,15 @@
 
   <modules>
     <module>robotpy</module>
-    <module>robotpy.feature</module>
+    
     <module>robotpy.updatesite</module>
+    
+    
+    <module>robotpy.feature</module>
```
Comment (virtuald) [inline pom.xml]: Hm? Why did this get moved around?

File: public.updatesite/build/update_version.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (virtuald) [inline public.updatesite/build/update_version.sh]: This is a duplicate of the script in /build, why?

File: robotpy/src/main/java/io/github/robotpy/plugins/robotpy/feature/feature.xml [hunk 3]
```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (virtuald) [inline robotpy/src/main/java/io/github/robotpy/plugins/robotpy/feature/feature.xml]: What is this file doing here? It already exists elsewhere.

File: pom.xml (hunk 1 above)
Comment (amboyscout) [reply pom.xml]: I think that was just an eclipse being odd thing. I didnt manually do that.

File: robotpy/src/main/java/io/github/robotpy/plugins/robotpy/feature/feature.xml (hunk 3 above)
Comment (amboyscout) [reply robotpy/src/main/java/io/github/robotpy/plugins/robotpy/feature/feature.xml]: I had some issues where there were files needed there that were actually in the robotpy.features folder. Copying them over fixed that issue.

File: public.updatesite/build/update_version.sh (hunk 2 above)
Comment (amboyscout) [reply public.updatesite/build/update_version.sh]: Oops, that was a change I forgot to ignore in github when I was working in eclipse. That can be removed.

**Pull request merged**